charts/leonardo/templates/cronjob.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: "resource-validator-cron"
```

Comment:
Can this be `leonardo-resource-validator-cron`?

In general, every K8s resource that is deployed via the leonardo chart should be prefixed with `leonardo`. All the app charts for an environment are deployed side-by-side in the same namespace (like `terra-dev`), and having a common prefix for a chart's resources reduces likelihood of conflicts

Sorry, we definitely need to document best practices like this somewhere :/